### PR TITLE
W-7411886 demo app

### PR DESCRIPTION
**Pull request by sf-vishwastyagi**

### CODE REVIEW

File: demo/mul4-einstein-connector-operations-demo/src/test/resources/log4j2-test.xml
Comment (ayush-salesforce) [inline demo/mul4-einstein-connector-operations-demo/src/test/resources/log4j2-test.xml:1]: As no tests are needed we can remove this file and folder

File: demo/mul4-einstein-connector-operations-demo/src/main/resources/application-types.xml
Comment (ayush-salesforce) [inline demo/mul4-einstein-connector-operations-demo/src/main/resources/application-types.xml:1]: Any usage of this file?

File: demo/mul4-einstein-connector-operations-demo/src/test/resources/log4j2-test.xml
Comment (sf-vishwastyagi) [reply demo/mul4-einstein-connector-operations-demo/src/test/resources/log4j2-test.xml:1]: done

File: demo/mul4-einstein-connector-operations-demo/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+* Anypoint Studio 7 with Mule ESB 4.6.9 Runtime.
+* MuleChain Einstein Connector v1.0.0
+
+
+How to Run Sample
+-----------------
+
+1. Import the project folder demo in Studio.
+2. Change the Client Id, Client Secret, Salesforce Org keys as per convenience in "envVars.json" file under src/main/resources folder.
```
[the remark this answers is not in the document]
Comment (sf-vishwastyagi) [reply demo/mul4-einstein-connector-operations-demo/README.md]: We don't need that

File: demo/mul4-einstein-connector-operations-demo/src/main/resources/application-types.xml
Comment (sf-vishwastyagi) [reply demo/mul4-einstein-connector-operations-demo/src/main/resources/application-types.xml:1]: This file is generated from anypoint studio. I think it is used by the studio

**Review by ayush-salesforce — APPROVED** [review]
LGTM

**Comment by tbolis-at-mulesoft** [comment]
LGTM

**Review by tbolis-at-mulesoft — APPROVED** [review]
Fine

**Pull request merged**